activejob/lib/active_job/log_subscriber.rb

```diff
@@ -57,8 +57,9 @@ def perform(event)
       job = event.payload[:job]
       ex = event.payload[:exception_object]
       if ex
+        cleaned_backtrace = Rails.backtrace_cleaner.clean(ex.backtrace)
```

Comment:
No Rails framework should have access to the `Rails` constant inside it since they can be used outside of a Rails application. We should be doing the same as `ActiveRecord::LogSubscriber`:

https://github.com/rails/rails/blob/40563a3ab457f5b1d8a5defaecb7375c55bc3fd5/activerecord/lib/active_record/railtie.rb#L97-L99
https://github.com/rails/rails/blob/40563a3ab457f5b1d8a5defaecb7375c55bc3fd5/activerecord/lib/active_record/log_subscriber.rb#L7

Reply:
Hey @rafaelfranca, thanks for taking a look.
I'll address this and fix the tests soon
Cheers!

Reply:
@rafaelfranca the changes that you suggested are made. CI seems to be failing on a flaky test. Is there a way for me to queue another build another than pushing some whitespace changes?

Reply:
@rafaelfranca tests passing. Lmk if there's anything else that I need to address here to get this merged!